packages/persistent-merkle-tree/src/node.ts

```diff
@@ -1,8 +1,21 @@
 import {HashObject} from "@chainsafe/as-sha256/lib/hashObject";
-import {hashObjectToUint8Array, hasher, uint8ArrayToHashObject} from "./hasher";
+import {executeHashComputations, hashObjectToUint8Array, hasher, uint8ArrayToHashObject} from "./hasher";
 
 const TWO_POWER_32 = 2 ** 32;
 
+export type HashComputation = {
+  src0: Node;
+  src1: Node;
+  dest: Node;
+};
```

Comment:
not sure this is the best name, other name could be `{left: Node; right: Node; parent: Node}`